src/alchemtest/amber/access.py

```diff
@@ -96,6 +97,10 @@ def load_invalidfiles():
         - 'data' : the example of invalid data files
         - 'DESCR': the full description of the dataset
 
+
+    .. deprecated:: 0.7
+        substituted by laod_testfiles
+
```

Comment:
At the start of the function, issue a DeprecationWarning
```python
warnings.warn("load_invalidfiles() was deprecated in 0.7.0 and will be removed in the following release. Use load_testfiles() instead", DeprecationWarning)
```

Reply:
Done!